WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/usecases/PostRestoreUseCase.kt

```diff
@@ -28,7 +28,7 @@ class PostRestoreUseCase @Inject constructor(
     suspend fun postRestoreRequest(
         rewindId: String,
         site: SiteModel,
-        types: RewindRequestTypes
+        types: RewindRequestTypes? = null
```

Comment:
Yeap, another good catch @malinajirka , you, you, you are good! 😄 

So, the reason why I decided to go with `null` instead of `RewindRequestType(true, true, ...)` is because the `RewindStatusService.rewind(...)` function is also doing the same, so I didn't want to change the behavior as I wasn't sure if that will end up being a functional change. See `val rewindResult = activityLogStore.rewind(RewindPayload(site, rewindId))`.

On the `FluxC` level passing `null` versus `types` is different, see `ActivityLogRestClient.rewind(...)` function and more specifically this block of code:
```
        val typesBody = if (types != null) {
            mapOf("types" to types)
        } else {
            mapOf()
        }
```

So on what case you would add an `empty map`, while on the other case you would add a `map of types`.

Wdyt?

Reply:
iirc, the empty map is acceptable in the API and is the default. I'll have to go back and test to be positive, that was so long ago.

Reply:
Yeah, I realize it'd slightly change the behavior but from my understanding the result will be the same. Having said that, let's not worry about that now and lets just stick to the nullable type. Thanks!